roles/systems/common/tasks/main.yaml

```diff
@@ -47,8 +47,9 @@
 #
 - name: Kill the unattended-upgrade task if it's running
   ansible.builtin.shell: |
-    killall -q -9 unattended-upgrade 2>&1 /dev/null
-  ignore_errors: true
+    if pgrep -x unattended-upgrade >/dev/null; then
+      killall -q -9 unattended-upgrade
+    fi
```

Comment:
This now fails if the kill does not work? Again, logic change here. Please explain